store/reducers/user.js

```diff
@@ -85,9 +79,9 @@ export default function (state = initialState.user, action) {
             user.preferences = user.data.website.preferences
           }
 
-          // Set anything we need in cookies for external access
-          Cookies.set('userId', user.id, dev ? { domain: '.fuelrats.com' } : {})
-          Cookies.set('allowUniversalTracking', user.preferences.allowUniversalTracking, dev ? { domain: '.fuelrats.com' } : {})
```

Comment:
I noticed you removed my universal tracking cookie. Add that back, otherwise it's going to cause conflicts with a change I have.

Reply:
Since we can store the object again, I went back to the preferences object.(see 2 lines below). Do we want to continue with JUST the allowUniversalTracking?

Reply:
Yeah, this is fine, then.